controller/MediaImport.php

```diff
@@ -45,12 +49,32 @@ public function index()
         parent::index();
     }
 
+    /**
+     * This action is called when requesting or submitting the upload form.
+     */
     public function editMedia()
     {
         $id = $this->hasRequestParameter('instanceUri')
             ? $this->getRequestParameter('instanceUri')
             : $this->getRequestParameter('id');
 
+        if (empty($id)) {
+            $this->returnError(self::ERR_NO_MEDIA_ID, true, 400);
```

Comment:
We can use the translated string, since it will be output to the enduser.

```suggestion
            $this->returnError(
                __('Request should provide a media identifier as the id or instanceUri parameter'), 
                true, 
                400
            );
```